spec/system/kudo_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'Kudo test' do
+  before do
+    driven_by(:rack_test)
+  end
+
+  it 'Creates, update and destroy kudo' do
+    employee = create(:employee)
+    visit root_path
+    click_link 'Sign In'
+    fill_in 'Email', with: employee.email
+    fill_in 'Password', with: employee.password
+    click_button 'Log in'
+    expect(page).to have_content 'Signed in successfully.'
```

Comment:
we shouldn't test login in kudo spec, we should be logged in here as written above

Reply:
That's right, it was an unnecessary step on my part

Reply:
That's right, it was an unnecessary step made by me